### ISSUE TITLE
[Need Discussion] Fix the size of image with fixed height and width

### ISSUE THREAD

**Issue by CleverFool77** [publiclab/plots2#4989]
## Description
I needed suggestion regarding this issue. I saw that the images does not have fixed width and height. If a user uploads a vertical image, it would not have uniformity with other images. 

![Screenshot from 2019-03-10 18-32-35](https://user-images.githubusercontent.com/26685258/54085754-577ccd00-4367-11e9-9a57-4d3ed0ffe8e4.png)

## Expected
We can fix this, by setting the fixed size of image width and height to let it remain uniform.

**Comment by CleverFool77** [comment]
If approved and is not considered as first-timer issue , then I would like to work on this.

**Comment by gauravano** [comment]
Hey @CleverFool77!

I think, we had this type of issue in the past. I tried to search but wasn't able to find much.
Let's discuss and close this - https://github.com/publiclab/plots2/issues/4120 one too.

The first image of your issue :point_down:

![Screenshot from 2019-03-10 18-32-35](https://user-images.githubusercontent.com/26685258/54085754-577ccd00-4367-11e9-9a57-4d3ed0ffe8e4.png)
 
is added via feature so only admins at publiclab.org can change this. 

The content in second image is from /blog I guess? so, I think we can find the code for that part in codebase. 

@jywarren can you cross-check the above details? Also, please let us know what do you think about this issue. Thanks!

**Comment by jywarren** [comment]
Hi! Actually those are set using the Features system (in the database) and we'll soon be redesigning the front page anyways. However just this kind of issue affects some other pages where we really need help!

https://publiclab.org/wiki/micro#Events and https://publiclab.org/wiki/coqui#Do+something+with+the+Coqui both show this behavior too; 

![image](https://user-images.githubusercontent.com/24359/54162409-bf94e700-442b-11e9-9c70-d9dd03ab735b.png)


but we have a clear CSS example of how to avoid it using CSS property `overflow:hidden`. See this page for how we've done this:

https://publiclab.org/research

![image](https://user-images.githubusercontent.com/24359/54162434-d89d9800-442b-11e9-90db-189170297265.png)

Note that we are OK with the images being cut off to make them fit. 

If you'd like to take another crack at this, we'd love it!

**Comment by jywarren** [comment]
Yes, though really in the example I gave on the dashboard/research page, we
are not actually cropping the images but hiding the extra edge of the image
to give them the appearance of being cropped. Awesome!
